README.md

```diff
@@ -37,15 +37,15 @@ You can deploy this app directly to Heroku to see the app live. Skip the setup i
 ### Running Unit Tests
 
 Unit tests do not make API calls to Braintree and do not require Braintree credentials. You can run this project's unit tests by
-calling `rake` on the command line.
+calling `rails -R spec` on the command line.
```

Comment:
We no longer using rake to manage these commands?

Reply:
The rake commands still work, I thought with the update to 5.2 we might want to adopt more rails-y command conventions since the [Rails CLI guides](https://guides.rubyonrails.org/command_line.html) are pretty well documented. I can push up a commit to include both options.

Reply:
Whatever is the best practice for rails development